### PR TITLE
Prod vs Dev updates

### PR DESCRIPTION
**Pull request by arb**
- Added context config to make paths shorter
- Added more prod vs dev branch logic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This is a basic react + webpack starter bundle. Be prepared to add your own style, this has less than none. The motivation for making this was we couldn't find a simple setup that was feature rich. We think this setup will scale nicely for small to medium sized web applications. If you're familier with webpack, you know it's very powerful, but sometimes lacking in the documentation and practical example departments. Hopefully this simple example will help you build your own setup or you can just start with ours!
 
+[![GitHub release](https://img.shields.io/github/release/continuationlabs/react-hot-reload-simple-starter.svg)](https://github.com/continuationlabs/react-hot-reload-simple-starter)
```
Comment (cjihrig) [inline README.md]: You're lead maintainer here, so it's your call. I like to put the badges right below the repo title, before the first paragraph of text. It's just less searching for the visual dashboard.

File: webpack.config.js [hunk 2]
```diff
@@ -24,15 +22,27 @@ if (argv.prod) {
   }));
   plugins.push(new Clean(['build']));
   plugins.push(new Webpack.optimize.OccurenceOrderPlugin(true));
+  plugins.push(new HtmlWebpackPlugin({
+    template: 'index.prod.template',
```
Comment (cjihrig) [inline webpack.config.js]: This doesn't exist, does it?

File: webpack.config.js [hunk 3]
```diff
@@ -24,15 +22,27 @@ if (argv.prod) {
...
+    inject: 'body'
+  }));
+}
+else {
+  entry.unshift('webpack/hot/dev-server');
+  plugins.push(new HtmlWebpackPlugin({
+    template: 'index.dev.template',
```
Comment (cjihrig) [inline webpack.config.js]: Same here. Do these get created at some point?

File: webpack.config.js (hunk 2 above)
Comment (arb) [inline webpack.config.js]: Nope, forgot to do `git add`

**Pull request merged**